server/routerlicious/packages/lambdas/src/scribe/lambda.ts

```diff
@@ -518,6 +533,9 @@ export class ScribeLambda implements IPartitionLambda {
 			// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
 			const message = this.pendingMessages.shift()!;
 			try {
+				if (!this.isDocumentCorrupt) {
+					throw new Error(`Test Error`);
```

Comment:
Do we want to remove this?

Reply:
@pradeepvairamani - I am still working on this, meant to mark it as a draft.